cms/djangoapps/contentstore/views/transcripts_ajax.py

```diff
@@ -264,7 +264,7 @@ def download_transcripts(request):
 
     # Construct an HTTP response
     response = HttpResponse(content, content_type=mimetype)
-    response['Content-Disposition'] = u'attachment; filename="{filename}"'.format(filename=filename.encode('utf-8'))
+    response['Content-Disposition'] = u'attachment; filename="{filename}"'.format(filename=filename)
```

Comment:
Was this encode statement added recently(to cater for py3 changes)? Looks good, but I think we should add a test just to be on the safe side.

Reply:
@DawoudSheraz This was probably missed during the python3 upgrade because it did not cause any test failures. Strings in python 3 are unicode by default so there is no need to encode it again. I am not sure what will we test in this case?